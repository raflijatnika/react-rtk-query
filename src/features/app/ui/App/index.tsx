const App = () => {
  return <h1>Entry Point</h1>
}

export { App }
