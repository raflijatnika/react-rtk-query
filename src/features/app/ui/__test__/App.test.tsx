// Test
import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer'

// UI
import { App } from '../App'

it('Component should render', () => {
  render(<App />)
  expect(screen.getByText('Hello World')).toBeInTheDocument()
})

it('Component should have snapshot', () => {
  const tree = renderer.create(<App />).toJSON()

  expect(tree).toMatchSnapshot()
})
