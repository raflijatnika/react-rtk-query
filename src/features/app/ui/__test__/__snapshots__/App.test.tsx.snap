// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should have snapshot 1`] = `
<h1>
  Hello World
</h1>
`;
